src/vs/base/common/fuzzyScorer.ts

```diff
@@ -871,18 +876,19 @@ export function prepareQuery(original: string): IPreparedQuery {
 				}
 
 				values.push({
-					original: originalPiece,
+					original: originalPiece.replace('*', '').replace('?', ''),
```

Comment:
```suggestion
					original: originalPiece?.replace('*', '')?.replace('?', ''),
```

Reply:
Thanks for taking a look at the code. But I don't see a need for using optional chaining here since originalPiece variable cannot be null nor undefined right ? I making the assumption based on the rest of the statements which use the same variable. Do you see otherwise ?